content/operate/kubernetes/logs/log-collector-rbac.md

```diff
@@ -0,0 +1,128 @@
+---
+Title: Log collector RBAC examples
+alwaysopen: false
+categories:
+- docs
+- operate
+- kubernetes
+description: YAML examples for configuring RBAC permissions for the Redis Enterprise log collector tool in restricted and all modes.
+linkTitle: Log collector RBAC
+weight: 50
+---
+
+This page provides YAML examples for configuring RBAC permissions for the Redis Enterprise log collector tool. The log collector requires different permission levels depending on the collection mode you choose.
+
+For complete log collection instructions, see [Collect logs]({{< relref "/operate/kubernetes/logs/collect-logs" >}}).
+
+## Prerequisites
+
+- [Deployment]({{< relref "/operate/kubernetes/deployment" >}}) must be installed
+- Appropriate permissions to create RBAC resources in target namespaces
+- Understanding of your deployment model (single namespace, multi-namespace, etc.)
+
+## Collection modes
+
+The log collector has two collection modes that require different RBAC permissions:
+
+- **Restricted mode** (recommended): Collects only Redis Enterprise resources with minimal security exposure. Default for versions 6.2.18-3 and later.
+- **All mode**: Collects comprehensive cluster information including nodes, storage classes, and operator resources. Use when specifically requested by Redis Support.
+
+## Restricted mode
+
+The restricted mode configuration provides minimal permissions for collecting Redis Enterprise resources only.
+
+{{<embed-yaml "k8s/log_collector_role_restricted_mode.md" "log-collector-restricted-rbac.yaml">}}
+
+Restricted mode configuration:
+- `Role`: Namespace-scoped permissions for Redis Enterprise resources
+- `ClusterRole`: Cluster-wide permissions for CRDs and basic cluster resources
+- `rules`: Minimal permissions for Redis Enterprise diagnostics
+
+Key permissions:
+- `pods, pods/log, pods/exec`: Access to pod information and logs
+- `app.redislabs.com/*`: All Redis Enterprise custom resources
+- `persistentvolumes`: Storage information for troubleshooting
+
+## All mode
+
+The all mode configuration provides comprehensive permissions for collecting detailed cluster information.
+
+{{<embed-yaml "k8s/log_collector_role_all_mode.md" "log-collector-all-rbac.yaml">}}
+
+All mode configuration:
+- `Role`: Extended namespace permissions including operator resources
+- `ClusterRole`: Additional cluster-wide permissions for nodes and storage
+- `rules`: Comprehensive permissions for full cluster diagnostics
+
+Additional permissions in all mode:
+- `nodes`: Node information and status
+- `storageclasses, volumeattachments`: Storage system details
+- `operators.coreos.com/*`: OpenShift operator information
+- `networking.istio.io/*`: Istio service mesh resources
+
+## Apply the configuration
+
+### Manual deployment
```

Comment:
These instructions only mention the roles, but it could also be useful to clarify that these roles should be bound (via RoleBindings) to the user subject that will execute the log collector.